graql/reasoner/type-generation.feature

```diff
@@ -167,6 +173,7 @@ Feature: Type Generation Resolution
 
   @ignore
   # TODO: re-enable when grakn#5824 is fixed (matching a variable with two types)
+    #  TODO: disabled because down and side casting no longer allowed
   Scenario: additional types for entities can be derived via attributes that they own
     Given for each session, graql define
       """
```

Comment:
There's a scenario later on in this file named `Scenario: additional types for relations can be derived using direct 'isa!'`. It actually isn't using type generation, so it shouldn't have been in this file in the first place. Let's rename it to `Scenario: a relation can be inferred based on a direct type` and move it to `relation-inference.feature` just under the scenario `Scenario: a rule can infer a relation based on ownership of any instance of a specific attribute type`.

Then, we can delete `type-generation.feature` entirely.

Reply:
Moved to where you suggested and deleted `type-generation.feature`